src/js/main.js

```diff
@@ -29,11 +30,11 @@ class Timeline extends React.Component {
 	render(){
 		let timelineResultClass;
 		let timelineResultOutput;		
-		if(this.state.error != null) {
+		if(_.isObject(this.state.error)) {
```

Comment:
this seems strange to me. Do you ever use the value of this.state.error?  What is the intended type